src/main/java/com/github/jpmand/intellijplugintest/toolWindow/MyToolWindowFactory.java

```diff
@@ -0,0 +1,51 @@
+package com.github.jpmand.intellijplugintest.toolWindow;
+
+import com.github.jpmand.intellijplugintest.services.MyProjectService;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.wm.ToolWindow;
+import com.intellij.openapi.wm.ToolWindowFactory;
+import com.intellij.ui.components.JBLabel;
+import com.intellij.ui.components.JBPanel;
+import com.intellij.ui.content.Content;
+import com.intellij.ui.content.ContentFactory;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+
+public class MyToolWindowFactory implements ToolWindowFactory {
+    private final ContentFactory contentFactory = ContentFactory.SERVICE.getInstance();
+    private static final Logger LOG = Logger.getInstance(MyToolWindowFactory.class);
+
+    public MyToolWindowFactory() {
+    }
+
+    @Override
+    public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
+        MyToolWindow myToolWindow = new MyToolWindow(toolWindow);
+        Content content = contentFactory.createContent(myToolWindow.getContent(), null, false);
+        toolWindow.getContentManager().addContent(content);
```

Comment:
add log here, like this:

```suggestion
        Content content = contentFactory.createContent(myToolWindow.getContent(), null, false);
        LOG.info("Got content")
        toolWindow.getContentManager().addContent(content);
```